config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_apikeyskeys.apikeys.cnrm.cloud.google.com.yaml

```diff
@@ -0,0 +1,249 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    cnrm.cloud.google.com/version: 0.0.0-dev
+  creationTimestamp: null
+  labels:
+    cnrm.cloud.google.com/managed-by-kcc: "true"
+    cnrm.cloud.google.com/stability-level: stable
+    cnrm.cloud.google.com/system: "true"
+    cnrm.cloud.google.com/tf2crd: "true"
+  name: apikeyskeys.apikeys.cnrm.cloud.google.com
+spec:
+  group: apikeys.cnrm.cloud.google.com
+  names:
+    categories:
+    - gcp
+    kind: APIKeysKey
+    plural: apikeyskeys
+    shortNames:
+    - gcpapikeyskey
+    - gcpapikeyskeys
+    singular: apikeyskey
+  preserveUnknownFields: false
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    - description: When 'True', the most recent reconcile of the resource succeeded
+      jsonPath: .status.conditions[?(@.type=='Ready')].status
+      name: Ready
+      type: string
+    - description: The reason for the value in 'Ready'
+      jsonPath: .status.conditions[?(@.type=='Ready')].reason
+      name: Status
+      type: string
+    - description: The last transition time for the value in 'Status'
+      jsonPath: .status.conditions[?(@.type=='Ready')].lastTransitionTime
+      name: Status Age
+      type: date
+    name: v1alpha1
```

Comment:
Since we are already manually adding it... Why not just mark the CRD as v1beta1? Do you see failing sample tests/integration tests when trying them locally? Or there are some fields with uncertainty?

IIUC this PR is basically doing the same as:

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.NewResourceFromTerraform.md

Reply:
This resource will only be as good as terraform, I think we have a higher bar for v1beta1?  (Maybe we didn't enforce this bar before we introduced v1alpha1?)